src/global/global_init.cc

```diff
@@ -165,13 +167,103 @@ void global_pre_init(
   g_conf().complain_about_parse_error(g_ceph_context);
 }
 
+#ifdef WITH_JAEGER
+  void global_pre_init(
```

Comment:
can you remove unrelated changes, it would really help in reviewing your work better + faster